content/en/blog/_posts/2022-03-31-Kubernetes-1-24-deprecations-and-removals

```diff
@@ -0,0 +1,90 @@
+---
+layout: blog
+title: "Kubernetes Removals and Deprecations In 1.24"
+date: 2022-03-31
+slug: upcoming-changes-in-kubernetes-1-24
+---
+
+**Author**: Mickey Boxell (Oracle)
+
+As Kubernetes evolves, features and APIs are regularly revisited and removed. New features may offer an alternative or improved approach to solving an existing problem, motivating the team to remove the old method. Rather than immediately being removed, old APIs are first deprecated and then removed according to the [Kubernetes Deprecation Policy](/docs/reference/using-api/deprecation-policy/). 
+
+We want to make sure you are aware of the changes coming in the Kubernetes 1.24 release. Kubernetes 1.24 is scheduled to be released in April. The release will **deprecate** several (beta) APIs that will become deprecated in favor of stable versions of the same APIs. The major change coming in the Kubernetes 1.24 release is the [removal of Dockershim](https://github.com/kubernetes/enhancements/issues/2221). This will be discussed below and in more depth at release time.  
+
+For an  early look at the changes coming in Kubernetes 1.24, take a look at the in-progress [CHANGELOG](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md).
+
+## The Kubernetes API removal and deprecation process
+
+Kubernetes contains a large number of components that evolve over time. In some cases, this evolution results in APIs, flags, or entire features, being removed. To prevent users from facing breaking changes, Kubernetes adopted a feature [deprecation policy](/docs/reference/using-api/deprecation-policy/). This policy ensures that stable APIs may only be deprecated when a newer stable version of that same API is available and that APIs have a minimum lifetime depending on their stability level: 
+
+* Generally available (GA) or stable API versions may be marked as deprecated but must not be removed within a major version of Kubernetes. 
+* Beta or pre-release API versions must be supported for 3 releases after deprecation. 
+* Alpha or experimental API versions may be removed in any release without prior deprecation notice. 
+
+Kubernetes will continue to remove APIs by following the deprecation policy regardless of whether an API is removed due to a beta feature graduating to stable or because that API was not proven to be successful and will also make sure migration options are documented. **Deprecated** APIs have been marked for removal in a future Kubernetes release. **Removed** APIs are those that are not longer available for use in current, supported Kubernetes versions after having been deprecated. These removals have been superseded by newer, stable/generally available (GA) APIs. 
```

Comment:
```suggestion
Kubernetes will continue to remove APIs by following the deprecation policy regardless of whether an API is removed due to a beta feature graduating to stable or because that API was not proven to be successful and will also make sure migration options are documented.  
**Deprecated** APIs have been marked for removal in a future Kubernetes release.  
**Removed** APIs are those that are not longer available for use in current, supported Kubernetes versions after having been deprecated. These removals have been superseded by newer, stable/generally available (GA) APIs. 
```
maybe spliting this in new lines makes sense here. i'm not sure how this would be rendered, tho